cxx/python/algorithms/basic_py.cc

```diff
@@ -108,6 +108,47 @@ PYBIND11_MODULE(basic, m) {
     py::arg("d"),
     py::arg("twin") )
   ;
+
+  m.def("_bundle_seed_data",&bundle_seed_data,
+    "Create SeismogramEnsemble from sorted TimeSeriesEnsemble",
+    py::return_value_policy::copy,
+    py::arg("d") )
+  ;
+
+  m.def("_BundleGroup",&BundleGroup,
+    "Bundle a seed grouping of TimeSeries into one or more Seismogram objects",
+    py::return_value_policy::copy,
```

Comment:
Why do we need to specify the return value policy here? Seems could be inefficient. @pavlis

Reply:
I was being conservative because I wasn't sure how this would behave.   BundleGroup does return a fairly large but for planned use normally not huge object (one or a few Seismogram objects).  My concern was memory management.   I was concerned that if we didn't copy the data it would vanish when BundleGroup when out of scope.   However, as a return I may be wrong about that.  This is deep in the subtleties of how python and C++ interact in this context.  In unknowns is why I intentionally made this copy.   I suspect my concern would not be difficult to test.   I could probably do so easily with the little python test program I hadn't put here because the data it was using is too huge for github.